src/sentry/snuba/metrics/fields/base.py

```diff
@@ -55,6 +60,21 @@
 )
 from sentry.utils.snuba import raw_snql_query
 
+__all__ = (
```

Comment:
Out of curiosity, why did you move this down?

Reply:
I didn't, pre-commit did